dcos/cosmospackage.py

```diff
@@ -451,31 +460,37 @@ def package_versions(self):
         return list(response.json().get("results").keys())
 
 
-def _get_header(request_type):
+def _get_header(request_type, version):
     """Returns header str for talking with cosmos
 
     :param request_type: name of specified request (ie uninstall-request)
     :type request_type: str
+    :param verison: version of request
+    :type version: str
     :returns: header information
     :rtype: str
     """
 
     return ("application/vnd.dcos.package.{}+json;"
-            "charset=utf-8;version=v1").format(request_type)
+            "charset=utf-8;version={}").format(request_type, version)
```

Comment:
I think we should instead construct all of the possible content type ahead of time. For example most RPCs in Cosmos are going to stay as `v1`. The type of this my could be: `Map[String, List[String]`, the key is the `request_type` and the value is a list of supported content-type in preference order. What do you think?